plugins/push/frontend/public/templates/common-components.html

```diff
@@ -261,20 +261,30 @@ <h5>{{ category }}</h5>
                 </el-select>
             </div>
             <div>
-                <details-tab-row :label="i18n('push-notification-details.message-title')">
-                    <template v-for="(component) in previewMessageTitle">
-                        <keep-alive>
-                            <component v-bind:is="component.name" :value="component.value"></component>
-                        </keep-alive>
-                    </template>
-                </details-tab-row>
-                <details-tab-row :label="i18n('push-notification-details.message-content')">
-                    <template v-for="(component) in previewMessageContent">
-                        <keep-alive>
-                            <component v-bind:is="component.name" :value="component.value"></component>
-                        </keep-alive>
-                    </template>
-                </details-tab-row>
+                <div class="bu-level bu-ml-5 bu-mb-4 bu-is-justify-content-flex-start bu-is-align-items-flex-start">
```

Comment:
unfortunately the default slot of details-tab-row component was being assigned falsy value even though there were components being passed. As a result, I ended up removing details-tab-row for this specific case only.